components/admin/invites/autonomous-simulation-page.tsx

```
@@ -0,0 +1,166 @@
+	const { state, setLoading, setSuccess, setError } = useAsyncState<StatusPayload>();
+	const router = useRouter();
+
+	const activeJobId = jobId ?? state.data?.runs[0]?.runId.split("-run-")[0];
+
+	useEffect(() => {
+		if (!jobId) {
+			return;
+		}
+		setLoading();
+		getAutonomousOnboardingSimulationStatus({ jobId }).then((response) => {
+			if (!response.success || !response.runs) {
```

Biome (2.4.4)

[error] Logical expression contains unnecessary complexity.

(lint/complexity/useSimplifiedLogicExpression)